packages/runner/lib/env.ts

```diff
@@ -1,9 +1,7 @@
 import { ENVS, parseEnvs } from '@nangohq/utils';
 import { getJobsUrl, getPersistAPIUrl } from '@nangohq/shared';
 
-export const envs = parseEnvs(ENVS.required({ RUNNER_NODE_ID: true }));
```

Comment:
not fully related but I have been wanting to do that for a long time. It should not required for enterprise setup